RecoTracker/MkFit/interface/MkFitHitWrapper.h

```diff
@@ -0,0 +1,60 @@
+#ifndef RecoTracker_MkFit_MkFitHitWrapper_h
+#define RecoTracker_MkFit_MkFitHitWrapper_h
+
+#include "DataFormats/Provenance/interface/ProductID.h"
+
+#include <memory>
+#include <vector>
+
+namespace mkfit {
+  class EventOfHits;
+  class TrackerInfo;
+  class Hit;
+  using HitVec = std::vector<Hit>;
+}  // namespace mkfit
+
+class MkFitHitWrapper {
+public:
+  MkFitHitWrapper();
+  MkFitHitWrapper(mkfit::TrackerInfo const& trackerInfo);
+  ~MkFitHitWrapper();
+
+  MkFitHitWrapper(MkFitHitWrapper const&) = delete;
+  MkFitHitWrapper& operator=(MkFitHitWrapper const&) = delete;
+  MkFitHitWrapper(MkFitHitWrapper&&);
+  MkFitHitWrapper& operator=(MkFitHitWrapper&&);
+
+  mkfit::EventOfHits& eventOfHits() { return *eventOfHits_; }
```

Comment:
what does it mean eventOfHits ? :(

Reply:
It's a container for LayerOfHits structures that mkFit uses to group (and index) hits. Having them grouped together allows mkFit to pass them easily around top-level steering functions.

Reply:
do you mind to add a comment ?

Reply:
I'll add a comment.

Reply:
Explanation added in new `MkFitEventOfHits` class.